contrib/awssqs/config/201-clusterrole.yaml

```diff
@@ -22,24 +22,24 @@ rules:
   resources:
   - deployments
   verbs:
-  - VerbAll
+  - *
 - apiGroups:
   - ""
   resources:
   - events
   verbs:
   - VerbAll
```

Comment:
You have another "VerbAll" here

Reply:
I did miss this one, thanks! I could not find any others.